scripts/ci-entrypoint.sh

```diff
@@ -152,6 +154,11 @@ create_cluster
 # export the target cluster KUBECONFIG if not already set
 export KUBECONFIG="${KUBECONFIG:-${PWD}/kubeconfig}"
 
+# install cloud-provider-azure components, if using out-of-tree
+if [[ -n "${TEST_CCM:-}" ]]; then
+  "${HELM}" install --repo https://raw.githubusercontent.com/kubernetes-sigs/cloud-provider-azure/master/helm/repo cloud-provider-azure --generate-name --set infra.clusterName="${CLUSTER_NAME}"
```

Comment:
how does this work when using a custom image of CCM and CNM that we just built above?

Reply:
whoops

Reply:
done, thanks for catching this!